src/components/search/instructors/dan-abramov/dan-abramov-page.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import {Meta, Story, Canvas} from '@storybook/addon-docs/blocks'
+import SearchDanAbromov from './index'
+
+<Meta title="components/search/instructors/dan-abramov" />
+
+# Curated Landing Page
+
+<Canvas>
+  <Story name="Default">
+    <div className="bg-gray-100 p-5">
+      <SearchDanAbromov
+        instructor={{
```

Comment:
this is the data the server provides, might as well use it

Reply:
Yes, this makes sense to me!